docs/musicfuel.md

```diff
@@ -0,0 +1,6 @@
+#musicfuel
+
+### Some basics:
```

Comment:
I like this line here, but I wonder if we may want to remove it for now since there aren't any other subsections yet.